applications/MetisApplication/tests/test_metis_submodelpart_list.py

```diff
@@ -0,0 +1,133 @@
+import os
+
+import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as kratos_utils
+
+if KratosMultiphysics.IsDistributedRun():
+    from KratosMultiphysics.mpi import distributed_import_model_part_utility
+
+def GetFilePath(fileName):
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), fileName)
+
+class TestMetisSubModelPartList(KratosUnittest.TestCase):
+    def setUp(self):
+        self.comm = KratosMultiphysics.DataCommunicator.GetDefault()
+        self.size = self.comm.Size()
+        self.rank = self.comm.Rank()
+
+    def tearDown(self):
+        with KratosUnittest.WorkFolderScope(self.work_folder, __file__):
+            self.comm.Barrier()
+            if self.rank == 0:
+                kratos_utils.DeleteFileIfExisting(self.file_name + ".time")
+            kratos_utils.DeleteFileIfExisting(self.file_name + "_" + str(self.rank) + ".mdpa")
+            self.comm.Barrier()
+
+    def ReadModelPart(self, model_part, settings):
+        if KratosMultiphysics.IsDistributedRun():
+            model_part.AddNodalSolutionStepVariable(KratosMultiphysics.PARTITION_INDEX)
+            model_part_import_util = distributed_import_model_part_utility.DistributedImportModelPartUtility(
+                model_part, settings)
+            model_part_import_util.ImportModelPart()
+            model_part_import_util.CreateCommunicators()
+        else:
+            import_flags = KratosMultiphysics.ModelPartIO.READ | KratosMultiphysics.ModelPartIO.SKIP_TIMER
+            mdpa_file_name = settings["model_import_settings"]["input_filename"].GetString()
+            KratosMultiphysics.ModelPartIO(mdpa_file_name, import_flags).ReadModelPart(model_part)

```

Comment:
please use [this](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python_scripts/testing/utilities.py#L14) instead

Reply:
this function does not allow to send my own settings

Reply:
can you try with my last commit?
Last request I promise :)

Reply:
Working fine now. You can have the last one ;)